src/main/Main.hs

```diff
@@ -63,6 +63,7 @@ import           Stack.Dot
 import           Stack.GhcPkg (findGhcPkgField)
 import qualified Stack.Nix as Nix
 import           Stack.FileWatch
+import           Stack.Freeze
```

Comment:
I think you forgot to `git add` the new modules.

Reply:
Ouch, forgot the main part of it